### PR TITLE
Applications REST Client Updates

### PR DESCRIPTION
**Pull request by winder**
Regenerate the clients and implement the new response unit tests.

### CODE REVIEW

File: src/main/java/com/algorand/algosdk/v2/client/model/TransactionApplication.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Fields for application transactions.
+ * Definition:
+ * data/transactions/application.go : ApplicationCallTxnFields
+ */
+public class TransactionApplication extends PathResponse {
+
+    // TODO: Generator needs to be updated to setup the Address like this.
```
Comment (winder) [inline src/main/java/com/algorand/algosdk/v2/client/model/TransactionApplication.java:20]: This was added by hand for now

File: src/main/java/com/algorand/algosdk/v2/client/model/DryrunApp.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package com.algorand.algosdk.v2.client.model;
```
Comment (algorandskiy) [inline src/main/java/com/algorand/algosdk/v2/client/model/DryrunApp.java:1]: this file should not be created, DryrunApp is gone and has no references

File: src/main/java/com/algorand/algosdk/v2/client/algod/TealDryRun.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package com.algorand.algosdk.v2.client.algod;
```
Comment (algorandskiy) [inline src/main/java/com/algorand/algosdk/v2/client/algod/TealDryRun.java:1]: this file should be deleted - see TealDryrun.java below

File: src/main/java/com/algorand/algosdk/v2/client/model/DryrunApp.java (hunk 2 above)
Comment (winder) [reply src/main/java/com/algorand/algosdk/v2/client/model/DryrunApp.java]: Thanks for catching this.

Comment (winder) [reply src/main/java/com/algorand/algosdk/v2/client/model/DryrunApp.java]: Looks like we're good now. I didn't see anything while grepping around for "DryRun" and "DryrunApp". Hopefully these are the only unused files that snuck in.

**Pull request merged**